### ISSUE TITLE
BSE & NSE codes

### ISSUE THREAD

**Issue by nbudihal** [Mittal-Analytics/Screener.in#42]
I would like to request column for displaying the BSE & NSE codes as a column option.

BTW the website is a great help for investors.

**Comment by vhku** [comment]
Yes, I agree. When we screen stocks, the first column is invariably Sl.No. (serial number). This might not be of great use to users (?). Instead of serial number, the BSE code for the stock could be displayed. That would be much more useful for those of us who copy the screen into Excel for further analysis, (for example historical prices etc).
